common/src/main/java/com/sequenceiq/cloudbreak/auth/altus/Crn.java

```diff
@@ -159,7 +159,7 @@ public static Partition safeFromString(String input) {
         FREEIPA("freeipa", NON_ADMIN_SERVICE),
         DATAHUB("datahub", NON_ADMIN_SERVICE),
         DATALAKE("datalake", NON_ADMIN_SERVICE),
-        DEX("dex", NON_ADMIN_SERVICE),
+        DE("de", NON_ADMIN_SERVICE),
```

Comment:
I have concerns about backwards compatibility. This will mean that you will need to synchronize the change for our internal accounts (I know DEX is not yet used on prod). I think it might be simpler to coordinate to add DE now and then remove DEX after everything has been moved to DE. I will let the DEX team decide what they want to do, since they know better than me if they are currently using DEX anywhere or if this change is to fix something which was already updated on the DEX side.

Reply:
@jamisonbennett good point. Yes this would mean we need to recreate any existing environments people may be using esp the ones used by QE team for testing. I can add DE as a new option here so that existing flows would continue to work. Once everyone migrates over raise another patch to remove DEX.